.cursor/hooks/post-edit-checks.sh

```
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+set -uo pipefail
+
+MAX_RETRIES=5
+PROJECT_HASH=$(echo "$PWD" | md5sum | cut -c1-12)
+RETRY_FILE="/tmp/konflux-ui-post-edit-retries-${PROJECT_HASH}"
+
+input=$(cat)
```

Shellcheck (0.11.0)

[warning] input appears unused. Verify use (or export if used externally).

(SC2034)